rclpy/rclpy/__init__.py

```diff
@@ -19,13 +19,18 @@
 from rclpy.exceptions import InvalidRCLPYImplementation
 from rclpy.impl import implementation_singleton
 from rclpy.impl import rmw_implementation_tools
+from rclpy.impl.rmw_implementation_tools import RCLPY_IMPLEMENTATION_ENV_NAME
 from rclpy.node import Node
+from rclpy.impl import excepthook
```

Comment:
Is there a reason for `excepthook` being imported at the end (instead of four lines above)? If yes, I think a comment would be helpful.